client/src/app/shared/components/projector/projector.component.ts

```diff
@@ -168,10 +168,14 @@ export class ProjectorComponent extends BaseComponent implements OnDestroy {
             this.offlineBroadcastService.isOfflineObservable.subscribe(isOffline => (this.isOffline = isOffline))
         );
 
-        this.slides = this.projectorSubject.pipe(
-            switchMap(projector => projector?.current_projections_as_observable || []),
-            map(projections =>
-                projections.map(
+        const trigger$ = merge(
```

Comment:
These changes have to be done to the presentation control component, too.

@tsiegleauq You can merge this PR if you want, but this is still to be done. I can't do it until next week

Reply:
i'll amend it tomorrow